test/sslapitest.c

```diff
@@ -757,7 +757,7 @@ static int test_tlsext_status_type(void)
 }
 #endif
 
-static int new_called = 0, remove_called = 0;
+static int new_called, remove_called, get_called;
```

Comment:
When adding the new ```get_called``` marker, one might ask if it is appropriate to check and clear it at *all* locations that currently do so for the existing markers, not just the ones that we expect to be relevant.

Reply:
I didn't do that because the get_cb is for server side only. I'm happy to make a follow-up PR, but IMO it doesn't add much value.

Reply:
Fair enough, and no need to make a follow-up PR just on my account.  Thanks for the explanation.